scripts/changelog.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
+
+import re
+import sys
+import os.path
+from pathlib import Path
+from typing import List, Optional
+
+from git_changelog import templates
```

Comment:
not from the built-ins, time to introduce a requirements.txt in the scripts dir?

Reply:
Yeah, actually this is a copy from https://github.com/RHEnVision/provisioning-backend/blob/main/scripts/changelog.py which is lazy. Perhaps we could create a new "pip module" and use it that way? I am not sure about terminology, I don't do much Python.

@ezr-ondrej creating a new git repo under the org and releasing it on pip should not be so hard I guess